### PR TITLE
Some bugs

### PR DESCRIPTION
**Pull request by ArtemidaOctavia**
all code get refactored with bnb standarts

### CODE REVIEW

File: src/content/effects.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import {
+  inventoryHolder, menu, gameplayUI, skillsHolder,
+} from '../utility/domElements';
+
+
+import { removeSkills } from '../utility/removeSkills';
+
+const effects = {
+  getRandomItem: (quantity) => {
+    for (let i = 1; i <= quantity; i += 1) {
```
Comment (copperkraft) [inline src/content/effects.js:19]: Don't combine business logic and dom manipulations. Follow the single responsibility principle.

File: src/content/effects.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        Object.keys(item.effects).forEach((effect) => {
+          effects[effect](item.effects[effect]);
+        });
+        if (item.consumable) {
+          effects.takeAwayItems([item]);
+        }
+      });
+    }
+    if (playerStatus.inventory.length > 15) {
+      while (playerStatus.inventory.length !== 15) {
```
Comment (copperkraft) [inline src/content/effects.js]: You can replace `if` + `while` with single `while` statement

File: src/content/effects.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        playerStatus.inventory.splice(-1);
```
Comment (copperkraft) [inline src/content/effects.js]: You can use `.pop()` to remove the last element of the array.

File: src/content/effects.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        inventoryHolder.removeChild(inventoryHolder.lastChild);
+      }
+    }
+  },
+  getSpecialItem: (specialItem) => {
```
Comment (copperkraft) [inline src/content/effects.js]: What 'Special' means?

File: src/content/effects.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    const item = specialItems[specialItem];
+    playerStatus.inventory.push(item.name);
+    const itemHolder = getDomElement('div', '', item.id, '', 'inventory-item');
```
Comment (copperkraft) [inline src/content/effects.js]: This code looks familiar. Consider refactoring.

File: src/content/effects.js (hunk 3 above)
Comment (copperkraft) [reply src/content/effects.js]: `.push()`, `.shift()`, `.unshift()` also exist.

File: src/content/effects.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  takeAwayAllItems: () => {
+    playerStatus.inventory = [];
+    killChildren(inventoryHolder);
+  },
+
+  moveTo: (id) => {
+    renderScene(id);
+  },
+  changeNeeds: (value) => {
+    playerStatus.health += value[0];
```
Comment (copperkraft) [inline src/content/effects.js]: Move all magic numbers to constants. Also, update stats based on input.

File: src/content/effects.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+      playerStatus.fatigue = 0;
+    }
+    if (playerStatus.hunger <= 0) {
+      playerStatus.hunger = 0;
+    }
+    if (playerStatus.thirst <= 0) {
+      playerStatus.thirst = 0;
+    }
+    changeStatusIndicators();
+    killPlayer();
```
Comment (copperkraft) [inline src/content/effects.js]: Why do you kill the player character after changing his stats?

File: src/content/effects.js [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+  },
+  moveForward: () => {
+    renderScene(getRandomDirection());
+  },
+  startNewGame: () => {
+    playerStatus.health = 100;
```
Comment (copperkraft) [inline src/content/effects.js]: Move starting values to constants.

File: src/content/scenes.js [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+        + 'незначительность причин, приводящих к смерти, выводили само понятие смерти за пределы человеческого '
+        + 'восприятия. Многие религии превращали смерть в божественную кару за греховное существование либо в '
+        + 'божественный дар, после которого человека ожидает счастливая и вечная жизнь (чаще всего понятие «смерть» '
+        + 'люди применяют по отношению к своему виду).',
+    image: 'https://i.ibb.co/485c7pc/youDied.jpg',
+    actions: {
+      newGame: {
+        actionDescription: 'Новая игра',
+        effects: [
+          { type: 'startNewGame', payload: '' },
```
Comment (copperkraft) [inline src/content/scenes.js]: You can omit payload if you don't use it.

File: src/content/effects.js [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+  inventoryHolder, menu, gameplayUI, skillsHolder,
+} from '../utility/domElements';
+import { getItemInDom } from '../utility/getItemInDom';
+import { getRandomKey } from '../utility/getRandomKey';
+import { items } from './items';
+import { getSkillInDom } from '../utility/getSkillInDom';
+import { resetStatus } from '../utility/resetStatus';
+
+const effects = {
+  getItem: ([quantity, specificItem]) => {
```
Comment (copperkraft) [inline src/content/effects.js:18]: Consider using the following data structure to determine "loot": 
```
{
  nut: [1, 2]  // for random quantity
  mushroom: 3 // for determined quantity
}

File: src/content/effects.js [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+    for (let i = 1; i <= quantity; i += 1) {
+      let item = getRandomKey(items);
+      while (item.cannotBeAttainedRandomly) {
+        item = getRandomKey(items);
+      }
+      if (specificItem) {
+        item = items[specificItem];
+      }
+      playerStatus.inventory.push(item.name);
+      getItemInDom(item);
```
Comment (copperkraft) [inline src/content/effects.js:28]: Did you mean `setItem`?

File: src/content/effects.js [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+  },
+  moveTo: (id) => {
+    renderScene(id);
+  },
+  changeNeeds: ([health, fatigue, hunger, thirst]) => {
+    playerStatus.health += health;
+    playerStatus.fatigue += fatigue;
+    playerStatus.hunger += hunger;
+    playerStatus.thirst += thirst;
+    if (playerStatus.health >= 100) {
```
Comment (copperkraft) [inline src/content/effects.js:50]: Move min and max values to constants.

File: src/content/effects.js [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+    itemsForTake.forEach(({ name, id }) => {
+      const itemIndex = playerStatus.inventory.indexOf(name);
+      playerStatus.inventory.splice(itemIndex, 1);
+      document.getElementById(id).remove();
+    });
+  },
+  moveTo: (id) => {
+    renderScene(id);
+  },
+  changeNeeds: ([health, fatigue, hunger, thirst]) => {
```
Comment (copperkraft) [inline src/content/effects.js:45]: Use an object instead of an array with defined length.

Comment (copperkraft) [reply src/content/effects.js:45]: You can use `Object.entries` iteration to update all stats.

File: src/utility/killPlayerIfExhausted.js [hunk 14]
```diff
@@ -0,0 +1,13 @@
+import { playerStatus } from '../content/playerStatus';
+import { renderScene } from './renderScene';
+
+const killPlayerIfExhausted = () => {
+  if (playerStatus.health <= 0
```
Comment (copperkraft) [inline src/utility/killPlayerIfExhausted.js:5]: Store critical values of stats in constant.

File: src/content/effects.js (hunk 11 above)
Comment (ArtemidaOctavia) [reply src/content/effects.js:28]: done

File: src/content/effects.js (hunk 12 above)
Comment (ArtemidaOctavia) [reply src/content/effects.js:50]: done

File: src/utility/killPlayerIfExhausted.js (hunk 14 above)
Comment (ArtemidaOctavia) [reply src/utility/killPlayerIfExhausted.js:5]: done

**Pull request merged**